site/src/components/Resources/Resources.tsx

```diff
@@ -103,40 +100,43 @@ export const Resources: FC<React.PropsWithChildren<ResourcesProps>> = ({
                       )}
 
                       <TableCell className={styles.agentColumn}>
-                        {agent.name}
-                        <div className={styles.agentInfo}>
-                          <span className={styles.operatingSystem}>{agent.operating_system}</span>
-                          {WorkspaceStateEnum[workspaceStatus] !==
-                            WorkspaceStateEnum["stopped"] && (
-                            <span style={{ color: agentStatus.color }} className={styles.status}>
+                      <TableCellDataPrimary highlight>{agent.name}</TableCellDataPrimary>
+                      <div className={styles.data}>
+                        <div className={styles.dataRow}>
+                          <strong>status: </strong>
```

Comment:
Just a convention on FE. We have a `Language` object that should be used to store texts in the UI.